### ISSUE TITLE
Tagging the topics manually and storing the map 

### ISSUE THREAD

**Issue by Mageswaran1989** [bmabey/pyLDAvis#89]
Is there any API/option that will allow user to select the topic and name it manually with an option to save this manual tagging.

Eg: 
- Load the pyLDAvis with data
- Adjust the lambda to say 0.6
- Select a topic say 1
- Manually pick a name for the topic by analysing the data
- Assign that to the current topic 
- Repeat this for all topics (optional: in the absence pick the top two words as topic)
- And save the mapping to a file for later reuse

**Comment by bmabey** [comment]
No, not at this time. I would be open to merging in something along these lines though.

**Comment by Mageswaran1989** [comment]
Looks like the changes needs to be done in https://cdn.rawgit.com/bmabey/pyLDAvis/files/ldavis.v1.0.0.js 

I will give it a shot!

**Comment by bmabey** [comment]
Yep, and then the python side of things would need to be able to take a dictionary of `topid id -> topic name` and pass it along to the JS.
